safe-global/safe-wallet-web
# Feat: persist last used order type in cowswap widget

## What it solves

Resolves: https://www.notion.so/safe-global/limit-etc-Remember-last-used-trade-type-6cd4087ac75947b2972e726653e32eb4?pvs=4

## How this PR fixes it
- Remember the last used trade type (swap or limit)
- persist across page visits.

## How to test it
- Change the trade type in the cow swap widget. (default is swap)
- Navigate away from the page and back again, or refresh the page. The trade type should stay as it was set before.

## Checklist
